GoMoney/Scences/Exchange/ExchangeCell.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+
+class ExchangeCell: UITableViewCell {
+    static let identifier = "exchange_cell"
+    lazy var symbolLabel = GMLabel(style: .largeBold, isCenter: true)
+    lazy var codeLabel = GMLabel(isCenter: true)
+    lazy var rateLabel = GMLabel()
+    lazy var amountLabel = GMLabel(style: .largeBold)
+
+    lazy var flag: UIImageView = .build { _ in }
+
```

Comment:
```suggestion
    lazy var flag: UIImageView = UIImageView()

```

Reply:
I use `.build` to set `translatesAutoresizingMaskIntoConstraints = false`